src/core/fixtures/settings/master.json

```diff
@@ -1209,28 +1209,6 @@
   "model": "core.setting",
```

Comment:
Really not a fan of fixtures. DB entries should really be created in the Unittest itself, not loaded as a fixture, but what the hell, it's how the original devs did it.

Reply:
Is this more of a preference or is there a good reason for this? 
(Unittest newbie so always keen to learn new things)